File: Programming-languages-and-laboratory/Python/Exercises/07_compito_java_20160129/Privato.py
from Cliente import Cliente


class Privato(Cliente):
    def __init__(self, tipo, codice, indirizzo, giorno, mese, anno, nome):
        super().__init__(tipo, codice, indirizzo, giorno, mese, anno)
        self._nome = nome
        
    def get_nome(self):
        return self._nome

    def __str__(self):
        return "Privato\t" + super().__str__() + "\t" + self._nome + "\t-\t" + self._indirizzo + "\t" + str(self._giorno) + "-" + str(self._mese) + "-" + str(self._anno) + "\t-"

File: Programming-languages-and-laboratory/Java/Exercises/07_compito_java_2016_01_29/Privato.java
import java.util.*;

public class Privato extends Cliente {

    private String nome;

    public Privato(int codice, String indirizzo, int giorno, int mese, int anno, String nome) {
        super(codice, indirizzo, giorno, mese, anno);
        this.nome = nome;
    }

    public String toString() {
        return "privato\t" + codice + "\t" + nome + "\t-\t" + indirizzo + "\t" + giorno + "-" + mese + "-" + anno + "\t-";
    }
}